course/templates/course/_course_cards.html

```diff
@@ -3,8 +3,7 @@
 
 {% for instance in instances %}
 {% with instance_url=instance|url %}
-<div class="col-sm-6 col-md-4 col-lg-3">
-	<div class="frontpage-course card">
+	<div class="frontpage course-card card">
```

Comment:
Is this course-card class used for anything? Or what is its intended purpose?

Reply:
That has become obsolete, you are correct. I removed that.